server/message_hooks.go

```diff
@@ -239,6 +239,16 @@ func (p *Plugin) SetChatReaction(teamsMessageID, srcUser, channelID string, emoj
 		return err
 	}
 
+	teamsMessage, err := client.GetChatMessage(chatID, teamsMessageID)
```

Comment:
I don't love this extra call needed, but I guess is the most robust approach. I would love to have a better way to do it. Anyway, this is not going to be a dramatic problem because the number of reactions should be relatively low, I guess.